tensorflow_datasets/image/coco_captions.py

```diff
@@ -0,0 +1,111 @@
+"""MSCOCO Captioning Dataset from 2014."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """
+@article{DBLP:journals/corr/LinMBHPRDZ14,
+  author    = {Tsung{-}Yi Lin and
+               Michael Maire and
+               Serge J. Belongie and
+               Lubomir D. Bourdev and
+               Ross B. Girshick and
+               James Hays and
+               Pietro Perona and
+               Deva Ramanan and
+               Piotr Doll{\'{a}}r and
+               C. Lawrence Zitnick},
+  title     = {Microsoft {COCO:} Common Objects in Context},
+  journal   = {CoRR},
+  volume    = {abs/1405.0312},
+  year      = {2014},
+  url       = {http://arxiv.org/abs/1405.0312},
+  archivePrefix = {arXiv},
+  eprint    = {1405.0312},
+  timestamp = {Mon, 13 Aug 2018 16:48:13 +0200},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/LinMBHPRDZ14},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+_DESCRIPTION = """
+COCO is a large-scale object detection, segmentation, and captioning dataset.
+COCO has several features:
+
+Object segmentation
+Recognition in context
+Superpixel stuff segmentation
+330K images (>200K labeled)
+1.5 million object instances
+80 object categories
+91 stuff categories
+5 captions per image
+250,000 people with keypoints
+
+In this dataset, there are 414113 captions, but 82783 images. Each image-caption
+pair is represented once in the dataset - thus, each image appears 5 times.
+"""
+
+_HOMEPAGE = "http://cocodataset.org/"
+
+class CocoCaptions(tfds.core.GeneratorBasedBuilder):
+  """MSCOCO Image Captioning Dataset - 2014"""
+
+  VERSION = tfds.core.Version('1.0.0')
```

Comment:
Please update version to 2.0.0

Reply:
Shouldn't the initial version be 1.0.0 (sorry if I'm misunderstanding, please let me know)?

Reply:
This is the tfds version, which should be 2.0.0 according to the mentors

Reply:
No, `1.0.0` is Ok. Actually for new datasets we start with `0.0.1`.

Reply:
@captain-pool told me to change to 2.0.0 on my additions, so I'm a bit contradicted haha.

Reply:
That's fine. It does make much more sense for it to be `1.0.0` or `0.0.1` - I changed it to the latter but I initially thought the former made more sense, since it's a new addition of data.

> If MAJOR version is incremented, the existing data has been changed and/or the slicing API doesn't necessarily return the same set of records for a given slice.